cmd/kubeadm/app/cmd/phases/init/certs.go

```diff
@@ -159,6 +159,9 @@ func getSANDescription(certSpec *certsphase.KubeadmCert) string {
 			// GetAPIServerAltNames errors without an AdvertiseAddress; this is as good as any.
 			AdvertiseAddress: "127.0.0.1",
 		},
+		NodeRegistration: kubeadmapiv1beta2.NodeRegistrationOptions{
+			CRISocket: kubeadmconstants.UnknownCRISocket, // avoid CRI detection
+		},
```

Comment:
this can be:
```
defaultConfig := cmdutil.DefaultInitConfiguration()
// GetAPIServerAltNames errors without an AdvertiseAddress; this is as good as any.
defaultConfig.LocalAPIEndpoint = kubeadmapiv1beta2.APIEndpoint{ AdvertiseAddress: "127.0.0.1 }
```